.vale/styles/config/vocabularies/docs/accept.txt

```diff
@@ -34,6 +34,7 @@ Grafana
 gsctl
 Honeybadger
 HTTP
+[Ii][Aa][Mm]
```

Comment:
```suggestion
IAM
```

is the only correct spelling

Reply:
then it triggers errors where `iam` appears (in the links and others. I need both sadly